compiler/llvm/clangUtil.cpp

```diff
@@ -2906,8 +2906,11 @@ void makeBinaryLLVM(void) {
   // libraries are written in C++. Here we use clang++ or possibly a
   // linker override specified by the Makefiles (e.g. setting it to mpicxx)
   std::string command = useLinkCXX + " " + options + " " +
-                        moduleFilename + " " + maino +
-                        " -o " + tmpbinname;
+                        moduleFilename + " " + maino;
+  if (fLinkStyle == LS_STATIC)
+    command += " -static";
+  command += " -o ";
+  command += tmpbinname;
```

Comment:
Thanks.  I steered clear of the library part because that is a separate project; there are other things that need to be done for that as well.

I've now added a comment that the dynamic flag is nothing, and the static flag is `-static` according to Makefile.clang.

Libmvec is only for gcc, so we don't have to worry about it here.